Project/Source/Components/UI/ComponentTransform2D.cpp

```diff
@@ -56,11 +82,59 @@ void ComponentTransform2D::OnEditorUpdate() {
 	float2 anchMin = anchorMin;
 	float2 anchMax = anchorMax;
 	ImGui::TextColored(App->editor->titleColor, "Anchors");
-	if (ImGui::DragFloat2("Min (X, Y)", anchMin.ptr(), App->editor->dragSpeed2f, 0, 1)) {
-		SetAnchorMin(anchMin);
+	if (ImGui::RadioButton("Custom", isCustomAnchor)) isCustomAnchor = true;
+	ImGui::SameLine();
+	if (ImGui::RadioButton("Presets", !isCustomAnchor)) {
+		isCustomAnchor = false;
+		anchorSelected = AnchorPreset::AnchorPresetType::MIDDLE_HORIZONTAL_CENTER_VERTICAL;
```

Comment:
Instead of setting to a default preset, the last selected preset could be remembered through some editor value (not stored) like lastAnchorPreset (?)

Reply:
Ummm...It's not a bad idea!
But I think is better put it in the middle center by default, only because to help the user to see at first sight the object in the center of the screen/viewport more than if the last preset is in the top right (for example) and the object move to that position.

Either is that viewport was too big that is impossible to see the object if it's in the top left (for example), but is easier to find if is in the middle of the viewport, i think.